ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-multicast.cfg

```diff
@@ -14,6 +14,7 @@ interface Management1
 !
 router multicast
    ipv4
+      counters rate period decay 300
```

Comment:
4.27.0F (cEOS) asks for `counters rate period decay 300 seconds`.

Reply:
I’ll double check.  I was using the latest M release for testing.

Reply:
This has been corrected.